### PR TITLE
[fix/update][l] Updating the scripts

### PR DESCRIPTION
**Pull request by gradedSystem**
## Changes made:

- Updated the scripts with a new function to update the byte size and column rows in `datapackage.json`
- Added `requirements.txt`
- Updated `scripts/README.md`

cc @anuveyatsu

### CODE REVIEW

File: scripts/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dataflows==0.5.5
```
Comment (anuveyatsu) [inline scripts/requirements.txt]: @gradedSystem are you actually using these libraries? I think you only have requests and xlrd.

Comment (gradedSystem) [reply scripts/requirements.txt]: @anuveyatsu Fixed

File: scripts/process.py [hunk 2]
```diff
@@ -1,32 +1,40 @@
...
 import csv
 import xlrd
+import json
+import datetime
+import requests
 
 def setup():
-	'''Crates the directorie for archive if they don't exist
+	'''
+        Crates the directorie for archive if they don't exist
```
Comment (anuveyatsu) [inline scripts/process.py]: @gradedSystem could you please fix this French to English, please? :smile:

File: scripts/process.py [hunk 3]
```diff
@@ -1,32 +1,40 @@
...
 	'''
 	if not os.path.exists('../archive'):
 		os.mkdir('../archive')
 
 def retrieve():
-    '''Downloades xls data to archive directory
+    '''
+        Downloades xls data to archive directory
```
Comment (anuveyatsu) [inline scripts/process.py]: @gradedSystem same here

File: scripts/process.py [hunk 4]
```diff
@@ -1,32 +1,40 @@
...
-    urllib.urlretrieve(source_monthly, monthly_dest)
+    response_monthly = requests.get(source_monthly)
+    with open(monthly_dest, 'wb') as monthly_file:
+        monthly_file.write(response_monthly.content)
     return daily_dest, monthly_dest
 
 def get_data(dest):
-    '''Gets the data from xls file and returns a ictionery of countries lists of it's data by year
+    '''
+        Gets the data from xls file and returns a ictionery of countries lists of it's data by year
```
Comment (anuveyatsu) [inline scripts/process.py]: Please fix the typo here @gradedSystem

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
@gradedSystem I left some minor comments.

File: scripts/process.py (hunk 2 above)
Comment (gradedSystem) [reply scripts/process.py]: Yeah, resolving that

**Comment by gradedSystem** [comment]
@anuveyatsu Fixed down the function names with the latest commit

**Pull request merged**